README.md

```diff
@@ -64,7 +64,15 @@ func testOutlets() {
 }
 ```
 
-#### Testing constraints
+### Testing UIKit Components
+#### UIControl
+
+```swift
+MTKAssertControl(testVC.loginButton, sends: #selector(LoginViewController.didTapLoginButton(_:)), to: testVC, for: .touchUpInside, "The login button should be hooked up to the login action.")
+ 
+```
+ 
+### Testing Constraints
```

Comment:
Hmm. Yea, that's a thought. It would be fairly different from the other things that would be added under UIKit Components, but it's definitely UIKit.